modules/sdk/gradle-plugins-workspace/src/gradleTest/clientExtensionValidateTask/testShouldThrowExceptionWhenInvalidTokenDefinitionFile.gradle

```diff
@@ -0,0 +1,22 @@
+import com.liferay.gradle.plugins.workspace.configurator.ClientExtensionProjectConfigurator
+
+task runGradleTest
+
+runGradleTest {
+	doLast {
+		String validateClientExtensionsTaskName = ClientExtensionProjectConfigurator.VALIDATE_CLIENT_EXTENSIONS_TASK_NAME
+
+		Project project = project(":client-extension:any-theme-css-project-invalid-token-definition-file")
+
+		Task validateTask = project.tasks[validateClientExtensionsTaskName]
+		try {
+			validateTask.actions.forEach {
+				it.execute(validateTask)
+			}
+			assert validateTask.state.failure != null
```

Comment:
This is a [different statement and the order matters](https://liferay.atlassian.net/wiki/spaces/ENGR/pages/1511752519/Java+conventions+in+Liferay#%5BinlineExtension%5DBlank-lines-separate-statements), there should be an empty line here before the assert